src/views/Playgrounds/PlaygroundUploadingFilesData.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { useState, useRef } from 'react'
+import { useUploadingFilesData } from '@/hooks'
+import styled from '@emotion/styled'
+import ImageCropDialog, { ImageCropDialogImperativeHandle } from '@/components/Dialogs/ImageCropDialog/ImageCropDialog'
+import { Avatar, Text, Button } from '@/shared/components'
+import { getVideoMetadata } from '@/utils/video'
+import { CropData } from '@/components/Dialogs/ImageCropDialog/cropper'
+
+export const PlaygroundUploadingFilesData = () => {
+  const {
+    uploadingFilesData,
+    addUploadingFileData,
+    updateUploadingFileStatus,
+    removeUploadingFileData,
+    clearUploadingFilesData,
+  } = useUploadingFilesData()
+  const [error, seterror] = useState<Error>()
+  const [avatarImageUrl, setAvatarImageUrl] = useState<string | null>(null)
+  const avatarDialogRef = useRef<ImageCropDialogImperativeHandle>(null)
+
+  const handleAvatarConfirm = (blob: Blob, url: string, cropData: CropData) => {
```

Comment:
Do you think we could also demonstrate restoring cropped image from the crop data here? So once we have the crop data saved, user should provide the same file and we should get the same cropped image without any additional user interaction. This will probably require some additional functionality in the cropper logic but should be useful later on